### ISSUE TITLE
Sticky tab - When using the command close other tabs a 2nd time the sticky tab closes

### ISSUE THREAD

**Issue by stephen147** [facelessuser/TabsExtra#63]
**To reproduce:**

1. Right-click and make the tab sticky.
2. Right-click on another tab and close other tabs.
3. The sticky tab doesn't close.
4. Right-click again on the tab in step 2 and close other tabs.
5. The sticky tabs now closes.

I've since tried: https://packagecontrol.io/packages/LockTab and it doesn't have this issue in case other people may want a solution.

**Comment by facelessuser** [comment]
Did you read the documentation? We mention that there is a "persistent" option https://facelessuser.github.io/TabsExtra/#sticky-tabs.

Originally, the option was added to quickly flag tabs you didn't want to close in the next bulk operation, and then once the bulk close happened, they returned to acting like normal tabs. That is the default. But if you wanted to make the persistent (as much as can be done with the Sublime API) you could enable the "persistent" option.

So basically, this works as intended and is not an issue.

@gir-bot remove S: triage
@gir-bot  add T: support

**Comment by stephen147** [comment]
Thanks, sorry for not noticing that.
